src/main/java/org/galatea/starter/service/IexService.java

```diff
@@ -45,5 +48,21 @@
     }
   }
 
+  /**
+   * Get the historical price for each symbol passed in for the date passed in.
+   *
+   * @param symbol list of symbols to get historical price for.
+   * @param date (formatted YYYYMMDD) the date for which you want the price on.
+   * @return a IexHistoricalPrices objects for the given symbols.
+   */
+  public List<IexHistoricalPrices> getHistoricalPricesForSymbol(
+      final String symbol, final String date) {
+    if (symbol.isEmpty() || date.isEmpty()) {
+      return Collections.emptyList();
```

Comment:
Should this be an empty list of an HTTP error? What do you think?
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status

Reply:
After our conversation yesterday, I went and updated both this and the lastTradedPrice to throw an error when an input is not provided. Wrote a small error handler and associated tests with it as well.